cookiecutter/main.py

```diff
@@ -72,15 +75,17 @@ def cookiecutter(
         password=password,
         directory=directory,
     )
+    import_patch = _patch_import_path_for_repo(repo_dir)
```

Comment:
This seems like it would add all the python files at the same level as the template directory to the import path. That might cause unexpected issues for people. Would it be better to limit the scope here to just the local extensions directory?

Reply:
While it makes some sense, I'm not sure it would be the best way to go. 
The general idea with the current implementation is that template extension loading will essentially continue to work the same way it always has, except that they will also be loaded from local sources. While the provided example shows extensions in the "local_extensions" folder, I'm not suggesting that the extensions need to be contained in that specific folder, or even that extensions should all exist in the same folder.

I'm not convinced that it's necessary to limit the scope, but I'll make the change if I haven't changed your mind. I assume that adding another config keyword doesn't cause any problems? In that case I would suggest changing this:

```
    {
        "project_slug": "Foobar",
        "year": "{% now 'utc', '%Y' %}",
        "_extensions": ["local_extensions.FoobarExtension"]
    }
```
to 
```
    {
        "project_slug": "Foobar",
        "year": "{% now 'utc', '%Y' %}",
        "_local_extensions_folders": ["local_extensions"],
        "_extensions": ["FoobarExtension"]
    }
```
any thoughts?